sql/core/src/test/scala/org/apache/spark/sql/DateFunctionsSuite.scala

```diff
@@ -477,7 +477,7 @@ class DateFunctionsSuite extends QueryTest with SharedSparkSession {
 
     checkAnswer(
       df.selectExpr("trunc(t, 'decade')"),
-      Seq(Row(Date.valueOf("2010-01-01")), Row(Date.valueOf("2010-01-01"))))
+      Seq(Row(null), Row(null)))
```

Comment:
can we create an individual test case for these unsupported fields? It's a bit weird to see some null tests randomly.